atlas/templates/Makefile.vars.gotmpl

```diff
@@ -3,15 +3,17 @@ BUILD_PATH              := bin
 DOCKERFILE_PATH         := $(CURDIR)/docker
 
 # configuration for image names
+BUILD_NUMBER            ?= 0
 USERNAME                := $(USER)
-GIT_COMMIT              := $(shell git describe --dirty=-unsupported --always --tags || echo pre-commit)
-IMAGE_VERSION           ?= $(GIT_COMMIT)-$(USERNAME)
+GIT_COMMIT              := $(shell git describe --dirty=-unsupported --always --tags --long || echo pre-commit)
```

Comment:
Is there a requirement to do the `--long` here?

Reply:
removed it.